news/5837.behavior.rst

```diff
@@ -0,0 +1,47 @@
+{% if top_line %}
```

Comment:
I don't understand the conditional logic in this news fragment ...

Reply:
Oh! I just took the template and replaced the text. I was a bit confused, too, since I didn't recall there being so much text last time. I suspect now that is not what I should have done. I can fix it.

Reply:
Just to be clear. What's the "standard" format for news file? I don't have my old one to refer to, and I assume using the template was incorrect.

P.S. Thank you for your patience with me!